### ISSUE TITLE
Focus ring went missing

### ISSUE THREAD

**Issue by SymMatt** [ChromeAngel/E4#6]
At some point the ring that highlights the focused object stopped appearing.

**Comment by ChromeAngel** [comment]
I think i found this.  The camera controller was referencing the focus and de-activating it when their was no selection... but not re-activating it.

The focus ring is now responsible for doing it's own thing and removed from the camera controller

Should be fixed in v0.3
